packages/symfony-static-dumper/src/FileSystem/FilePathResolver.php

```diff
@@ -46,9 +46,8 @@ public function resolveFilePathWithArgument(Route $route, string $outputDirector
         }
         $i = 0;
         return Strings::replace($filePath, self::FILE_PATH_REGEX, function () use (&$i, $arguments) {
-            $value = $arguments[$i];
-
             ++$i;
+            $value = $arguments[$i];
```

Comment:
it seems false positive

Reply:
I temporary skip `MoveVariableDeclarationNearReferenceRector` on `FilePathResolver.php`